mulesoft-ai-chain/1.0/modules/ROOT/pages/configuring-tools-operations.adoc

```diff
@@ -0,0 +1,200 @@
+= Configuring Tools Operations
+
+== Before You Begin
+
+To use Tools you must create a `tools.config.json` file that includes all the necessary API information for the operation to execute APIs successfully, for example:
+
+[source,json]
+----
+[
+    {
+        "action": "Execute POST requests for API endpoints.",
+        "url": "http://check-inventory-material.cloudhub.io/mmbe",
+        "name": "Check Inventory",
+        "method": "POST",
+        "headers": "Basic XXX",
+        "example-payload": "{\n \"materialNo\": \"paramValue\"}",
+        "query": [
+            "Check inventory in SAP",
+            "Check Stock Overview",
+            "Show product availability for MULETEST0",
+            "Check the inventory for MULETEST0",
+            "Check Stock for 400-110"
+        ],
+        "description": "Check inventory details for material in SAP by providing the materialNo as input for paramValue. Please use the materialNo and not materialNumber. This action applies whenever users' intent is 'Stock overview', 'product availability', 'Inventory', 'available stock'. Use the headers to perform the request."
+    },
+    {
+        "action": "Execute GET requests for API endpoints.",
+        "url": "https://anypoint.mulesoft.com/mocking/api/v1/sources/exchange/assets/612eb739-4266-4f4c-bf2f-29953c153d80/accounts-api/1.0.1/m/accounts",
```

Comment:
Wondering if we shouldn't be including this URL as is in our documentation...?

Reply:
@rstastny9 I'm not sure what your comment means. You mean we shouldn't include this URL in the documentation?